src/index.html

```diff
@@ -11,7 +11,174 @@
+          type="date"
+          name="DateOfBirth"
+          class="info"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement<input type="checkbox"
+          name="terms"
+          class="info"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset name="secondBlock" class="block">
+        <legend>Registration</legend>
+
+        <label class="formField">
+        E-mail: <input
+        type="email"
+        placeholder="email@example.com"
+        name="email"
+        required
+        class="info"
+        >
+        </label>
+
+        <label class="formField">
+          Password: <input
+          type="password"
+          name="Password"
+          required
+          minlength="5"
+          maxlength="25"
+          class="info"
+        >
+        </label>
+
+      </fieldset>
+
+      <fieldset name="thirdBlock" class="block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="formField">
+        Do you love cats?
+
+          <label class="inline formField">
+            <input
+            type="radio"
+            name="lowesCats"
+            class="info"
+            > Yes
+          </label>
+
+          <label  class="inline formField">
+            <input
+            type="radio"
+            name="lowesCats"
+            class="info"
+            > No
+          </label>
+
+        </div>
+
+        <label class="formField">
+          What is your favorite color? <input
+          type="color"
+          name="FavoriteColor"
+          class="info"
+          >
+        </label>
+
+        <label class="formField">
+          What time do you go to bed? <input
+          type="time"
+          name="TimeToBadAt"
+          class="info"
+          >
+        </label>
+
+        <label class="formField">
+          What are you favorite brand of cars?
+          <select name="theFavoriteModelOfCar" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="formField">
+          How do you rate our work? <input
+          type="range"
+          name="RateOfWork"
+          value="0"
+          class="info"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset name="fourthBlock" class="block">
+        <legend>Additional info:</legend>
+
+        <label class="formField">
+        Comments:
+          <textarea ></textarea>
+        </label>
+
+        <label class="formField" name="Recommendation">
+          Would you recommend us?
+          <select class="info">
+            <option>yes</option>
+            <option>no</option>
+          </select>
+        </label>
+
+      </fieldset>
+
+      <button
+      name="fifthBlock"
+      type="submit"
+      class="block"
+      >
+      Submit
+      </button>
```

Comment:
fix indentation here also
```suggestion
      <button
        name="fifthBlock"
        type="submit"
        class="block"
      >
        Submit
      </button>
```